css/about.css

```diff
@@ -0,0 +1,632 @@
+    box-shadow: 0.7px 0.7px 1px white;
+    background-color: white;
+  }
+
+  /***** Headline *****/
+
+  #headline {
+    width: 100%;
+    background-image: url('https://summit.cckorea.org/images/base/main_big.png');
+    background-size: cover;
+    background-position: 0% -3em;
+    height: 140vh;
+  }
+
+  .text-container {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    top: -2em;
+    width: 60%;
+    margin: auto;
+  }
+
+  .welcome {
+    margin-bottom: 0;
+    font-size: 1.5em;
+  }
+
+  .main-header {
+    display: flex;
+    align-items: center;
+  }
+
+  .h1_1 {
+    position: relative;
+    top: -0.2em;
+    font-size: 2.5em;
+    font-weight: 900;
+  }
+
+  .h1_2 {
+    position: relative;
+    top: -0.6em;
+    font-size: 2.5em;
+    font-weight: 900;
+  }
+
+  .description {
+    font-size: 0.7em;
+    width: 70%;
+    padding: 0.8em;
+    line-height: 1.5em;
+    border: 2px solid var(--primary);
+    background-color: white;
+  }
+
+  .contact {
+    font-size: 0.7em;
+  }
+
+  .link {
+    text-decoration: underline;
+    color: #272a31;
+  }
+
+  /***** Logo *****/
+
+  .about-logo-container {
+    width: 60%;
+    margin: auto;
+  }
+
+  .about-logo-header {
+    color: var(--footer);
+    font-size: 1em;
+  }
+
+  .logo-img {
+    width: 40%;
+  }
+
+  #about-logo-head {
+    display: flex;
+    top: -1em;
+    gap: 0.4em;
+    justify-content: center;
+    flex-direction: row;
+  }
+
+  #about-logo-head::after {
+    left: 48%;
+  }
+
+  .logo-text {
+    position: relative;
+    display: block;
+    top: -2.5em;
+    line-height: 1.5em;
+    font-size: 0.7em;
+    text-align: center;
+    color: var(--footer);
+    margin-bottom: 1em;
+  }
+
+  .logo {
+    width: 60%;
+  }
+
+  /***** History *****/
+
+  .card {
+    display: flex;
+    justify-content: space-between;
+    width: 70%;
+    margin: auto;
+    flex-direction: row;
+  }
+
+  .history-header {
+    color: var(--footer);
+    font-size: 1em;
+  }
+
+  .history-sub-header {
+    font-size: 0.8em;
+    color: var(--footer);
+  }
+
+  .history-header::after {
+    left: 47%;
+  }
+
+  .card1,
+  .card2 {
+    width: 49%;
+    height: 10em;
+    background-size: cover;
+    position: relative;
+  }
+
+  .card-cover {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+  }
+
+  .see {
+    font-size: 0.8em;
+  }
+
+  /***** Partner *****/
+
+  .sponsors {
+    flex-direction: row;
+    align-items: center;
+    width: 70%;
+    display: flex;
+    margin: auto;
+    position: relative;
+  }
+
+  .sponsors1 {
+    margin: 0;
+    width: 60%;
+  }
+
+  .sponsors2 {
+    gap: 1em;
+    width: 40%;
+  }
+
+  .partners-header {
+    font-size: 1em;
+    color: var(--primary);
+  }
+
+  .partners-header::after {
+    left: 48%;
+  }
+
+  /***** footer *****/
+
+  #footer {
+    background-color: var(--primary);
+    padding: 1em 4em;
+    justify-content: center;
+  }
```

Comment:
- Hey, let's improve the background color or the footer on the about page to be close to the original design on the desktop version. See the image below:


*DESIGN*
![Screenshot from 2023-03-17 17-57-36](https://user-images.githubusercontent.com/59397159/225970925-757e5350-4c0a-4029-9b2e-426052d65f90.png)

*your design*
![Screenshot from 2023-03-17 17-57-13](https://user-images.githubusercontent.com/59397159/225970931-ae722638-7d1d-4801-97d5-ddebd8a70ce9.png)